### PR TITLE
Fixing Editor console resize #822, #622

### PR DESCRIPTION
**Pull request by erangatl**
## Purpose
Fixing Editor console resize #822, #622

### CODE REVIEW

File: components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/js/output-console/console-list.js [hunk 1]
```diff
@@ -86,8 +86,15 @@ define(['log', 'jquery', 'lodash', 'backbone', 'console'], function (log, $, _,
             showConsoleComponents: function () {
                 var self = this;
                 var consoleHeaderContainer = self._$parent_el;
-                $('#service-tabs-wrapper').css('height', '65%');
+                var serviceWrapper =  $('#service-tabs-wrapper');
+                serviceWrapper.css('height', '65%');
                 consoleHeaderContainer.removeClass('hide');
+                $("#service-tabs-wrapper").resizable({
```
Comment (TanyaM) [inline components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/js/output-console/console-list.js]: Shall we stick to single quotes in js files without using both.

File: components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/js/output-console/service-console-list-manager.js [hunk 2]
```diff
@@ -64,6 +64,7 @@ define(['log', 'jquery', 'lodash', 'output_console_list', 'workspace', 'service_
                         if (this.isActive()) {
                             this._activateBtn.parent('li').removeClass('active');
                             this.hideAllConsoles();
+                            $( "#service-tabs-wrapper" ).resizable( "destroy" );
```
Comment (TanyaM) [inline components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/js/output-console/service-console-list-manager.js]: single quotes

File: components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/js/output-console/console-list.js (hunk 1 above)
Comment (erangatl) [reply components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/js/output-console/console-list.js]: Fixed with #1690

File: components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/js/output-console/service-console-list-manager.js (hunk 2 above)
Comment (erangatl) [reply components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/js/output-console/service-console-list-manager.js]: Fixed with #1690

**Pull request merged**